sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MapStateImpl.scala

```diff
@@ -98,6 +112,7 @@ class MapStateImpl[K, V](
     StateStoreErrors.requireNonNullStateValue(key, stateName)
     val compositeKey = stateTypesEncoder.encodeCompositeKey(key)
     store.remove(compositeKey, stateName)
```

Comment:
We probably want to document that the number does not represent the "actual" state rows to delete?

Reply:
Done